fast64_internal/sm64/sm64_f3d_writer.py

```diff
@@ -1,12 +1,69 @@
-import shutil, copy, bpy, cProfile, pstats
-
-from ..panels import SM64_Panel
-from ..f3d.f3d_writer import *
-from ..f3d.f3d_material import *
-from ..f3d.f3d_gbi import FMaterial
-from .sm64_texscroll import *
-from .sm64_utility import *
+import shutil, copy, bpy, re, os
+from io import BytesIO
+from math import ceil, log, radians
+from mathutils import Matrix, Vector
```

Comment:
I'm personally not a fan of several module imports on the same line (`import shutil, copy, ...`)
nor importing specific things from modules with short and recognizable names (`from math import ceil, ...`)

But it's a style thing that doesn't affect much in the end so I don't mind too much. It's okay like this to me